pkg/ca/kmsca/kmsca_test.go

```diff
@@ -36,7 +37,7 @@ func TestNewKMSCA(t *testing.T) {
 
 	chain := []*x509.Certificate{subCert, rootCert}
 
-	ca, err := NewKMSCA(context.WithValue(context.TODO(), fake.KmsCtxKey{}, subKey), "fakekms://key", chain)
+	ca, err := NewKMSCA(context.WithValue(context.TODO(), fake.KmsCtxKey{}, subKey), "fakekms://key", chain, crypto.SHA256)
```

Comment:
I would like to add a test for this, but that would require further code changes.